stages/05-release/README.md

```diff
@@ -9,8 +9,11 @@ The production ready artefact is promoted to the production environment.
 ## Steps
 
 1. deployment to production environment
+    * [deployment strategyes](https://docs.openshift.com/container-platform/3.3/dev_guide/deployments/deployment_strategies.html)
```

Comment:
Please take latest platform version: https://docs.openshift.com/container-platform/latest/applications/deployments/deployment-strategies.html

Reply:
Updated the link